internal/multigitter/run.go

```diff
@@ -307,6 +311,10 @@ func (r *Runner) runSingleRepo(ctx context.Context, repo scm.Repository) (scm.Pu
 		return nil, errors.Wrap(err, "could not push changes")
 	}
 
+  return r.ensurePullRequestExists(ctx, repo, prRepo, baseBranch, featureBranchExist)
+}
+
+func (r *Runner) ensurePullRequestExists(ctx context.Context, repo scm.Repository, prRepo scm.Repository, baseBranch string, featureBranchExist bool) (scm.PullRequest, error) {
```

Comment:
`log` has lost all context in this function. The best thing to do it probably to include it as a parameter as well.